src/modules/mavlink/mavlink_receiver.cpp

```diff
@@ -427,12 +335,10 @@ MavlinkReceiver::send_flight_information()
 {
 	mavlink_flight_information_t flight_info{};
 
-	param_t param_flight_uuid = param_find("COM_FLIGHT_UUID");
+	uint64_t flight_uuid = static_cast<uint64_t>(_p_flight_uuid.get());
 
-	if (param_flight_uuid != PARAM_INVALID) {
-		int32_t flight_uuid;
-		param_get(param_flight_uuid, &flight_uuid);
-		flight_info.flight_uuid = (uint64_t)flight_uuid;
```

Comment:
@dager, I cast it as uint64_t because that's how it is used later.  I'll revert the PARAM_INVALID error I created.

Reply:
I've reverted it, but I still think the way I had it, casting as a uint64_t at the param_t.get() call saved a copy.  Maybe I'm not seeing things clearly?